packages/clay-panel/src/index.tsx

```diff
@@ -120,7 +136,7 @@ const ClayPanel: React.FunctionComponent<IProps> & {
 								collapsed: showIconCollapsed,
 							}
 						)}
-						onClick={animate}
+						onClick={() => setExpanded(!expanded)}
```

Comment:
Why not call `animate` here in conjunction with `setExpanded`?

Reply:
We want the ability to animate the component whenever the `expand` prop is changed. We can add animate here, but that doesn't allow us to actually animate it by changing the prop from the parent component